.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,4 +1,14 @@
-# Feature Request - Name
+---
+name: Feature Request
```

Comment:
Request New Model or Feature?

Reply:
I called this issue template Feature Request because it deals with model zoo infrastructure. I've added a separate model_request.md file for new model requests.